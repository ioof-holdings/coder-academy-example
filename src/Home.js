import React from "react";
import logo from "./logo.svg";
import "./Home.css";

const Home = () => (
  <div data-testid="home" className="Home">
    <header className="Home-header">
      <img src={logo} className="Home-logo" alt="logo" />
      <p>
        Edit <code>src/App.js</code> and save to reload.
      </p>
      <a
        className="Home-link"
        href="https://reactjs.org"
        target="_blank"
        rel="noopener noreferrer"
      >
        Learn React
      </a>
    </header>
  </div>
);

export default Home;
