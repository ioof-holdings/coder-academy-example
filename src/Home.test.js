import React from "react";
import { render, cleanup } from "@testing-library/react";
import Home from "./Home";

afterEach(cleanup);

test("should render home component", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
