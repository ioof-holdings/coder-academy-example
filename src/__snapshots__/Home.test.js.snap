// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home component 1`] = `
<div>
  <div
    class="Home"
    data-testid="home"
  >
    <header
      class="Home-header"
    >
      <img
        alt="logo"
        class="Home-logo"
        src="logo.svg"
      />
      <p>
        Edit 
        <code>
          src/App.js
        </code>
         and save to reload.
      </p>
      <a
        class="Home-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</div>
`;
